### PR TITLE
style(frontend): new header title when signed off [GIX-3015]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

New header title when the user is not signed in.

Screenshot 2024-09-27 at 10 45 31
Screenshot 2024-09-27 at 10 45 49
Screenshot 2024-09-27 at 10 45 56

### CODE REVIEW

File: src/frontend/src/lib/i18n/en.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
 	},
 	"auth": {
 		"text": {
-			"title_part_1": "A wallet that lives on-chain for",
-			"title_part_2": "100% decentralization",
+			"title_part_1": "Multi-chain wallet.",
+			"title_part_2": "100% onchain",
```
Comment (DenysKarmazynDFINITY) [inline src/frontend/src/lib/i18n/en.json:46]: On the screenshot that Andronikos shared in Slack, the second part also has a dot (`100% on chain.`).

**Review by DenysKarmazynDFINITY — APPROVED** [review]
The code LGTM, just one comment re. the translation.

File: src/frontend/src/lib/i18n/en.json (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/lib/i18n/en.json:46]: The foundation wants "onchain" I understand.

See for example this recent PR applied to the portal: https://github.com/dfinity/portal/pull/3530

Comment (DenysKarmazynDFINITY) [reply src/frontend/src/lib/i18n/en.json:46]: Oh true, it should be `100% onchain.`. The question about the dot is still pending though.

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/i18n/en.json]: > On the screenshot that Andronikos shared in Slack, the second part also has a dot (100% on chain.).

Good catch! Corrected

**Pull request merged**